of-builder/Dockerfile

```diff
@@ -1,15 +1,15 @@
-FROM golang:1.9-alpine AS builder
+FROM golang:1.10-alpine AS builder
```

Comment:
Is this version move required for the change?

Reply:
Build fails with 1.9
```
Step 6/11 : RUN go build -o /usr/bin/of-builder .
 ---> Running in b7782fb9892d
# github.com/alexellis/of-cloud/of-builder/vendor/github.com/docker/docker/pkg/archive
vendor/github.com/docker/docker/pkg/archive/archive.go:364:5: hdr.Format undefined (type *tar.Header has no field or method Format)
vendor/github.com/docker/docker/pkg/archive/archive.go:364:15: undefined: tar.FormatPAX
vendor/github.com/docker/docker/pkg/archive/archive.go:1166:7: hdr.Format undefined (type *tar.Header has no field or method Format)
vendor/github.com/docker/docker/pkg/archive/archive.go:1166:17: undefined: tar.FormatPAX
The command '/bin/sh -c go build -o /usr/bin/of-builder .' returned a non-zero code: 2
```

If 1.10 is problematic we would need to vendor older version of docker/docker (and buildkit as well maybe)

Reply:
This container is going to be self-contained so it may not be so much of an issue - but the package with issues in 1.10 (tar / os/user ) is used in other functions. https://github.com/golang/go/issues/24787